### DISCUSSION TITLE
Welcome to fake-filler-extension Discussions!

### DISCUSSION THREAD

**Discussion by anburocky3** [anburocky3/fake-filler-extension#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hi @anburocky3, My name is **Shajid Shafee**, and I'm from Sri Lanka. As a Full stack engineer, I would like to contribute to this development environment, and also gain knowledge from the industry experts.
